File: fizzbuzz.rb
#!/bin/ruby

(1..100).each { |i|
    if i % 3 == 0
        print "Fizz"
    end
    if i % 5 == 0
        print "Buzz"
    end
    if i % 3 != 0 and i % 5 != 0
        print i
    end
    puts ""
}

File: fizzbuzz.lua
for i=1,100 do
	if (i % 3 == 0) then
		io.write("Fizz")
	end
	if (i % 5 == 0) then
		io.write("Buzz")
	end
		if (i % 3 ~= 0 and i % 5 ~= 0) then
		io.write(i);
	end
	io.write("\n");
end
